java/migration/csv/pom.xml

```diff
@@ -53,7 +52,7 @@
         <dependency>
             <groupId>ai.grakn</groupId>
             <artifactId>client-java</artifactId>
-            <version>${grakn.version}</version>
+            <version>1.5</version>
```

Comment:
@sorsaffari package name is no longer `ai.grakn:client-java`, please see graknlabs/client-java#2

Reply:
added the correct dependencies